src/routes/tools/annual-averages/_store.js

```diff
@@ -184,11 +184,9 @@ export const bookmark = derived(
     const { location, boundaryId } = $locationStore;
     const { imperial } = $unitsStore;
     const [lng, lat] = location.center;
-    const bookmark = `climvar=${$climvarStore}&scenario=${$scenarioStore}&models=${$modelsStore}
-    &imperial=${imperial}&lng=${lng}&lat=${lat}&boundary=${boundaryId}`;
-    if (process.browser) {
-      return `${window.location.href}?${bookmark}`;
+    if (boundaryId === "custom") {
+      return "Cannot create a bookmark for an uploaded boundary";
     }
-    return null;
+    return `climvar=${$climvarStore}&scenario=${$scenarioStore}&models=${$modelsStore}&imperial=${imperial}&lng=${lng}&lat=${lat}&boundary=${boundaryId}`;
```

Comment:
It would be good to make this change, but I need to think about the best way to do this. Currently when a tool loads, I use reverse gecoding since I only have the lat, lng. When you search for a location and update chart, I use forward geocoding. So the results returned from the 2 different MapBox endpoints are different. This is an issue only for LOCA grid cells, other boundaries wont have this issue since the names are from the Cal-Adapt API.

Reply:
Ah okay, that makes sense. Seems like an edge case then but a potentially confusing one for users. We can log it in Trello and come back to it later if you don't want to tackle it now.

Reply:
Made a note of the [issue](https://trello.com/c/i6nQgTF3/66-make-location-name-retrieved-from-mapbox-consistent-for-the-same-grid-cell) in Trello.

Reply:
Thank you!